client/my-sites/pages/page/index.js

```diff
@@ -670,6 +682,25 @@ class Page extends Component {
 		this.props.recordEvent( 'Clicked Copy Page' );
 	};
 
+	exportPage = () => {
+		this.props.recordEvent( 'Clicked Export Page' );
+		const { page } = this.props;
+
+		const fileContent = JSON.stringify( {
+			__file: 'wp_template',
+			language: 'en',
+			title: page.title,
+			content: page.content,
+		} );
+		const blob = new window.Blob( [ fileContent ], { type: 'application/json' } );
+		const link = document.createElement( 'a' );
+		link.href = URL.createObjectURL( blob );
+		link.download = page.title + '.json';
+		document.body.appendChild( link );
+		link.click();
+		document.body.removeChild( link );
```

Comment:
Better use `saveAs()` helper, example:

https://github.com/Automattic/wp-calypso/blob/de18db59f34af488ff47092e28b596ad9dade68a/client/my-sites/stats/stats-download-csv/index.jsx#L58-L60

Reply:
Oh! TIL! I'll update the code. Thanks!!!